src/main/java/com/epam/ta/reportportal/core/widget/content/filter/ActivityFilterStrategy.java

```diff
@@ -38,6 +44,19 @@
 	@Override
 	public Map<String, ?> buildFilterAndLoadContent(LoadContentStrategy loadContentStrategy, ReportPortalUser.ProjectDetails projectDetails,
 			Widget widget) {
-		return null;
+		UserFilter userFilter = widget.getFilter();
+		Filter filter = new Filter(userFilter.getTargetClass(), Sets.newHashSet(userFilter.getFilterCondition()));
+		filter = updateWithDefaultConditions(filter, projectDetails.getProjectId());
+		return loadContentStrategy.loadContent(widget.getContentFields(), filter, widget.getWidgetOptions(), widget.getItemsCount());
+	}
+
+	private Filter updateWithDefaultConditions(Filter filter, Long projectId) {
```

Comment:
as i see, this method is used in every filter strategy
will it be better to think about moving it in basic interface and call something like 'specifyDefaultConditions'